GViewCore/include/GView.hpp

```diff
@@ -192,7 +192,7 @@ namespace Hashes
       public:
         bool Init(CRC32Type type);
         bool Update(const unsigned char* input, uint32 length);
-        bool Update(Buffer buffer);
+        bool Update(const Buffer& buffer);
         bool Final(uint32& hash);
         static std::string_view GetName(CRC32Type type);
         const std::string GetHexValue();
```

Comment:
Asa generic la functiile astea (GetHexValue).
Avand in vedere ca size-ul in HEX e fix in functie de hash (e.g. CRC32 = 8 caractere) --> nu ar fi mai comod sa ai un char temp[<size fix>] definit in fiecare clasa si sa nu mai faci alocari si sa returnezi un string_view ?

Reply:
am adaugat direct un LocalString<256>